src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
+// Defining this global in .jshintrc would create a danger of using the global
+// unguarded in another place, it seems safer to just disable jsHint for these
+// three lines.
+/* jshint ignore: start */
+if ( typeof Symbol === "function" && Symbol.iterator ) {
```

Comment:
Do we need to worry about cases where a global Symbol is defined as a
function and Symbol.iterator doesn't have a typeof equal to "symbol"? I
imagine this would break using this code with Symbol polyfills that can't
override typeof.

-- 
Michał Gołębiowski